Celeste.Mod.mm/Mod/Registry/DecalRegistry.cs

```diff
@@ -17,6 +17,11 @@ public static class DecalRegistry {
             { "parallax", delegate(Decal decal, XmlAttributeCollection attrs) {
                 ((patch_Decal)decal).MakeParallax(float.Parse(attrs["amount"].Value));
             }},
+            { "scale", delegate(Decal decal, XmlAttributeCollection attrs) {
+                float scalex = attrs["multiplierX"] != null ? float.Parse(attrs["multiplierX"].Value) : 1f;
```

Comment:
might want to rename to `multiplyX` & `multiplyY`

Reply:
yeah I think those sound better